apps/organizations/templates/organizations/cms/organization.html

```diff
@@ -2,14 +2,13 @@
 {% load cms_tags %}
 
 {% block content %}
+  <h1>{{ current_page.get_title }}</h1>
 
-<h1>{{ current_page.get_title }}</h1>
+  {% with current_page.organizationpage.organization as organization %}
+    {% if organization.logo %}
+      <img src="{{ organization.logo.url }}">
```

Comment:
If you don't mind, can you add an `alt` attribute here?

Reply:
Done